test/test.js

```diff
@@ -1,41 +1,70 @@
 var mocha = require('mocha');
 
-var Movie = require('../database/index.js').Movie;
+// var db = require('../database/index.js');
+const data = require('../database/data.js');
 
-const request = require('supertest');
+const chai = require('chai');
 
-const mongoose = require('mongoose');
+const expect = chai.expect;
 
+const mongoose = require('mongoose');
 
 //Connect to MongoDB Here
 
-var testMovie = new Movie({
-  id: 2,
-  url: 'test_url',
-  thumbnail: 'test_thumbnail',
-  category: 'test category',
-  videoId: 4,
-  title: 'test title',
-  author: 'test author',
-  plays: 3
-});
+var testModel = mongoose.model('testModel', new mongoose.Schema({
+  id: Number,
+  url: String,
+  thumbnail: String,
+  category: String,
+  videoId: Number,
+  title: String,
+  author: String,
+  plays: Number
+})); 
 
-mongoose.connect('mongodb://localhost/testdb')
-  .then(() => {
-    console.log('connected to db');
-    testMovie.save((err) => {
-      if (err) {
-        console.log('there was an error', err);
-      } else {
-        console.log('saved it ');
-      }
-    });
-    
+before(function (done) {
+  //Connect to MongoDB Here
+  mongoose.connect('mongodb://localhost/testdb');
+  mongoose.connection.once('open', function () {
+    console.log('Connected to MongoDB!');
+    done();
+  }).on('error', function () {
+    console.log('Connection error : ', error);
   });
+});
+ 
+beforeEach(function (done) {
+  testModel.deleteMany({}, (err) => {
+    if (err) {
+      console.log('there was an error', err);
+    }
+    done();
+  });
+});
 
 
-
-
-
-
-
+describe('seeding function', () => {
+  it('should add 20 entries (movies) into database', (done) => {
+    
```

Comment:
These are nit pick suggestions, but I think that you delete line 48 and bring the body of this test up the newline doesn't add as much readability as you might think. Same thing goes for lines 44-45 you can probably bring the spacing down to one newline between each block of code.

Reply:
That's a good idea. Gonna implement that